### ISSUE TITLE
Using wrong device id will return integer 0 instead of string from serial_number and device_name attributes

### ISSUE THREAD

**Issue by Kazhuu** [piotrbulinski/flexit_bacnet#10]
If you run this code 
```python
import asyncio
from flexit_bacnet import FlexitBACnet

async def main():
    device = FlexitBACnet('192.168.1.11', 3) # device id = 3
    await device.update()

    print('Device Name:', device.device_name) # will return 0
    print('Serial Number:', device.serial_number) # will return 0

if __name__ == "__main__":
    asyncio.run(main())
```

According to function type hints these should only return `str`. How this should be fixed? I could also take a stab at this if needed.

**Comment by piotrbulinski** [comment]
@Kazhuu would returning an empty string on that occasion solve it?

**Comment by Kazhuu** [comment]
I think that will but is this even supposed to be a valid connection with wrong device id? I'm not that familiar with BACnet protocol to know this. If so, could this already be checked earlier when the connection is made for the first time and some error thrown or something similar? If it's supposed to be valid connection, then this is also fine. Maybe this should be mentioned in the README as well?

**Comment by Kazhuu** [comment]
I guess this solution also solves the issue @piotrbulinski. With this change I only need to compare to the empty string instead of integer.

**Comment by piotrbulinski** [comment]
@Kazhuu I think that's the best for now. I've released version 2.1.1 with that fix.

**Comment by Kazhuu** [comment]
Thanks!
